src/DayColumn.js

```diff
@@ -140,6 +153,45 @@ class DaySlot extends React.Component {
     );
   }
 
+  renderAvailabilities = () => {
+    let {
+      availabilities,
+      availabilityComponent,
+      availabilityWrapperComponent: AvailabilityWrapper,
+      availabilityStartAccessor,
+      availabilityEndAccessor,
+      min,
+      date,
+    } = this.props;
+    let AvailabilityComponent = availabilityComponent;
+    let styledAvailabilities = getStyledAvailabilities({
+      availabilities, availabilityStartAccessor, availabilityEndAccessor, min, totalMin: this._totalMin, date
+    });
+
+    return styledAvailabilities.map(({availability, style}, index) => {
+      let { height, top } = style;
+
+      return AvailabilityWrapper && (
+        <AvailabilityWrapper key={index}>
```

Comment:
Is there a better key we can use than index? `renderEvents` below uses an `entityKeyAccessor` if available, otherwise it falls back to an index key.